lib/hammer_cli_katello/host_errata.rb

```diff
@@ -46,10 +46,9 @@ class InfoCommand < HammerCLIKatello::ErratumInfoCommand
     end
 
     class RecalculateCommand < HammerCLIKatello::SingleResourceCommand
-      include HammerCLIForemanTasks::Async
       resource :host_errata, :applicability
       command_name "recalculate"
-      success_message _("Errata recalculated with task %{id}.")
+      success_message _("Errata recalculated.")
```

Comment:
I think different wording may be appropriate here.  This is a past-tense message, while in reality the recalculation may still be in progress when the user sees this.  Perhaps something a bit more accurate like
```suggestion
      success_message _("Errata recalculation started.")
```
cc @jturel to get his thoughts as well

Reply:
seems like calling `success_message` is the standard way to do this, based on its 104 existing uses in this repo I'm okay with this.

Reply:
+1 for eventually getting sucess messages from the server, but def out of the scope of this pr to remove the 104 use cases.

Reply:
Message updated.